packages/mobile/src/components/lineup-tile/LineupTileTopRight.tsx

```diff
@@ -77,25 +85,52 @@ type Props = {
    * Whether or not to show the artist pick icon
    */
   showArtistPick?: boolean
+  /**
+   * Premium conditions to determine what icon and label to show
+   */
+  premiumConditions?: Nullable<PremiumConditions>
 }
 
 export const LineupTileTopRight = ({
   duration,
   isArtistPick,
   isUnlisted,
-  showArtistPick
+  showArtistPick,
+  premiumConditions
 }: Props) => {
+  const { neutralLight4 } = useThemeColors()
+  const accentBlue = useColor('accentBlue')
   const trackTileStyles = useTrackTileStyles()
 
   return (
     <View style={styles.topRight}>
-      {showArtistPick && isArtistPick && (
-        <LineupTileTopRightItem icon={IconStar} label={messages.artistPick} />
+      {!!premiumConditions && (
+        <LineupTileTopRightItem
```

Comment:
It's top left on web but top right on mobile? Haven't checked figma but ok.

Reply:
this is different than the corner tag. it's a label. web also has it on top right